can't run the v3.0.1 tests against an installed copy of seqan

## Platform

  - SeqAn version: 3.0.1
  - Operating system: Linux mrc-tux 5.3.0-26-generic #28-Ubuntu SMP Wed Dec 18 05:37:46 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
  - Compiler: g++ (Debian 9.2.1-23) 9.2.1 20200110

## Description

### How to repeat the problem

Build and install the libseqan3-dev version 3.0.1 Debian package
and run this script: https://salsa.debian.org/med-team/seqan3/blob/master/debian/tests/run-tests from the root of the source package

### Expected behaviour
Tests build and execute successfully

### Actual behaviour

```
[  7%] Building CXX object alignment/matrix/CMakeFiles/edit_distance_trace_matrix_full_test.dir/edit_distance_trace_matrix_full_test.cpp.o
cd /tmp/autopkgtest.ZZbmhm/autopkgtest_tmp/build_unit/alignment/matrix && /usr/bin/ccache /usr/bin/c++  -DSEQAN3_HAS_BZIP2=1 -DSEQAN3_HAS_ZLIB=1 -I/tmp/autopkgtest.ZZbmhm/autopkgtest_tmp/build_unit/vendor/googletest/googletest/include -I/tmp/autopkgtest.ZZbmhm/autopkgtest_tmp/unit/SEQAN3_CLONE_DIR-NOTFOUND/test/include -isystem /usr/include/seqan3/submodules/sdsl-lite/include  -O3 -DNDEBUG   -pedantic -Wall -Wextra -Werror -std=c++17 -fconcepts -pthread -o CMakeFiles/edit_distance_trace_matrix_full_test.dir/edit_distance_trace_matrix_full_test.cpp.o -c /tmp/autopkgtest.ZZbmhm/autopkgtest_tmp/unit/alignment/matrix/edit_distance_trace_matrix_full_test.cpp
/tmp/autopkgtest.ZZbmhm/autopkgtest_tmp/unit/alignment/matrix/edit_distance_trace_matrix_full_test.cpp:11:10: fatal error: seqan3/test/pretty_printing.hpp: No such file or directory
   11 | #include <seqan3/test/pretty_printing.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/test/seqan3-test.cmake
+++ b/test/seqan3-test.cmake
@@ -42,7 +42,12 @@ file(MAKE_DIRECTORY ${SEQAN3_TEST_CLONE_DIR}/googletest/include/)
 add_library (seqan3_test INTERFACE)
 target_compile_options (seqan3_test INTERFACE "-pedantic"  "-Wall" "-Wextra" "-Werror")
 target_link_libraries (seqan3_test INTERFACE "seqan3::seqan3" "pthread")
-target_include_directories (seqan3_test INTERFACE "${SEQAN3_CLONE_DIR}/test/include/")
+if (SEQAN3_CLONE_DIR)
+    target_include_directories (seqan3_test INTERFACE "${SEQAN3_CLONE_DIR}/test/include/")
+else ()
+    target_include_directories (seqan3_test INTERFACE "${CMAKE_CURRENT_LIST_DIR}/include/")
+endif ()
+
 add_library (seqan3::test ALIAS seqan3_test)
 
 # seqan3::test::performance specifies required flags, includes and libraries

--- a/test/seqan3-test.cmake
+++ b/test/seqan3-test.cmake
@@ -25,6 +25,8 @@ option (SEQAN3_TEST_BUILD_OFFLINE "Skip the update step of external projects." O
 # Paths to folders.
 # ----------------------------------------------------------------------------
 
+find_path (SEQAN3_TEST_INCLUDE_DIR NAMES seqan3/test/tmp_filename.hpp HINTS "${CMAKE_CURRENT_LIST_DIR}/include/")
+
 set (SEQAN3_BENCHMARK_CLONE_DIR "${PROJECT_BINARY_DIR}/vendor/benchmark")
 set (SEQAN3_TEST_CLONE_DIR "${PROJECT_BINARY_DIR}/vendor/googletest")
 
@@ -42,12 +44,7 @@ file(MAKE_DIRECTORY ${SEQAN3_TEST_CLONE_DIR}/googletest/include/)
 add_library (seqan3_test INTERFACE)
 target_compile_options (seqan3_test INTERFACE "-pedantic"  "-Wall" "-Wextra" "-Werror")
 target_link_libraries (seqan3_test INTERFACE "seqan3::seqan3" "pthread")
-if (SEQAN3_CLONE_DIR)
-    target_include_directories (seqan3_test INTERFACE "${SEQAN3_CLONE_DIR}/test/include/")
-else ()
-    target_include_directories (seqan3_test INTERFACE "${CMAKE_CURRENT_LIST_DIR}/include/")
-endif ()
-
+target_include_directories (seqan3_test INTERFACE "${SEQAN3_TEST_INCLUDE_DIR}")
 add_library (seqan3::test ALIAS seqan3_test)
 
 # seqan3::test::performance specifies required flags, includes and libraries

--- a/test/unit/argument_parser/detail/CMakeLists.txt
+++ b/test/unit/argument_parser/detail/CMakeLists.txt
@@ -1,4 +1,4 @@
-find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}" "${seqan3_test_unit_BINARY_DIR}/..")
+find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}")
 
 add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${SEQAN3_TEST_LICENSE_DIR}")
 seqan3_test(format_help_test.cpp)

--- a/test/unit/argument_parser/detail/CMakeLists.txt
+++ b/test/unit/argument_parser/detail/CMakeLists.txt
@@ -1,4 +1,7 @@
-add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${seqan3_test_unit_BINARY_DIR}/..")
+# seqan3_test_unit_BINARY_DIR is needed for debian builds
+find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}" "${seqan3_test_unit_BINARY_DIR}/..")
+
+add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${SEQAN3_TEST_LICENSE_DIR}")
 seqan3_test(format_help_test.cpp)
 seqan3_test(format_html_test.cpp)
 seqan3_test(format_man_test.cpp)

--- a/test/unit/argument_parser/detail/CMakeLists.txt
+++ b/test/unit/argument_parser/detail/CMakeLists.txt
@@ -1,4 +1,4 @@
-add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_INCLUDE_DIR}")
+add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${seqan3_test_unit_BINARY_DIR}/..")
 seqan3_test(format_help_test.cpp)
 seqan3_test(format_html_test.cpp)
 seqan3_test(format_man_test.cpp)

--- a/test/unit/argument_parser/detail/CMakeLists.txt
+++ b/test/unit/argument_parser/detail/CMakeLists.txt
@@ -1,4 +1,3 @@
-# seqan3_test_unit_BINARY_DIR is needed for debian builds
 find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}" "${seqan3_test_unit_BINARY_DIR}/..")
 
 add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${SEQAN3_TEST_LICENSE_DIR}")

--- a/test/unit/argument_parser/detail/format_help_test.cpp
+++ b/test/unit/argument_parser/detail/format_help_test.cpp
@@ -275,7 +275,7 @@ TEST(help_page_printing, copyright)
     const char * argvCopyright[] = {"./copyright", "--copyright"};
     argument_parser copyright("myApp", 2, argvCopyright);
 
-    std::ifstream license_file{std::string{{SEQAN_INCLUDE_DIR}} + "/../LICENSE.md"};
+    std::ifstream license_file{std::string{{SEQAN3_TEST_LICENSE_DIR}} + "/LICENSE.md"};
     std::ranges::subrange<std::istreambuf_iterator<char>, std::istreambuf_iterator<char>> sub
     {
         std::istreambuf_iterator<char>(license_file),

